dubbo-registry/dubbo-registry-api/src/main/java/org/apache/dubbo/registry/integration/RegistryProtocol.java

```diff
@@ -122,8 +122,8 @@
 
     private final static Logger logger = LoggerFactory.getLogger(RegistryProtocol.class);
     private static RegistryProtocol INSTANCE;
-    private final Map<URL, NotifyListener> overrideListeners = new ConcurrentHashMap<>();
-    private final Map<String, ServiceConfigurationListener> serviceConfigurationListeners = new ConcurrentHashMap<>();
+    private final Map<URL, List<NotifyListener>> overrideListeners = new ConcurrentHashMap<>();

```

Comment:
I am not convinced that we need a list of overrideListeners and ServiceConfigurationListener here. I will leave this to @chickenlj to judge.

Reply:
what's the difference between `serviceKey` classified listeners and `URL` classified listeners?
 Map<String, ServiceConfigurationListener> serviceConfigurationListeners 
Map<URL, List<NotifyListener>> overrideListeners